apps/a11y-tests/package.json

```diff
@@ -4,11 +4,12 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "build": "node ../../scripts/just.js build-commonjs-only",
+    "build": "npx just-scripts build",
```

Comment:
I think the templates themselves need to be changed... Consulted with @jdhuntington a bit about this point. It probably is clearer to point at the script that it is actually calling rather than the indirectly called thing...